### ISSUE TITLE
Undefined names: 'ident' and 'start'

### ISSUE THREAD

**Issue by cclauss** [hanxiao/bert-as-service#5]
[flake8](http://flake8.pycqa.org) testing of https://github.com/hanxiao/bert-as-service on Python 3.7.1

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./service/server.py:176:44: F821 undefined name 'ident'
                    worker.send_multipart([ident, b'', pickle.dumps(self.result)])
                                           ^
./service/server.py:178:55: F821 undefined name 'start'
                    time_used = time.perf_counter() - start
                                                      ^
./service/server.py:180:46: F821 undefined name 'ident'
                                (num_result, ident, time_used, int(num_result / time_used)))
                                             ^
./bert/tokenization.py:40:31: F821 undefined name 'unicode'
        elif isinstance(text, unicode):
                              ^
./bert/tokenization.py:63:31: F821 undefined name 'unicode'
        elif isinstance(text, unicode):
                              ^
5     F821 undefined name 'unicode'
5
```

**Comment by hanxiao** [comment]
thanks will fix it

**Comment by hanxiao** [comment]
oh btw are you a bot?

**Comment by cclauss** [comment]
Nope.

**Comment by hanxiao** [comment]
fixed in #6
